Essentials/src/config.yml

```diff
@@ -512,6 +512,10 @@ npcs-in-balance-ranking: false
 # This is useful when a sign sells or buys one item at a time and the player wants to sell a bunch at once.
 allow-bulk-buy-sell: true
 
+# Allow selling of named items with the /sell command
+# This is useful so people don't sell custom items
```

Comment:
```suggestion
# Allow selling of items with custom names with the /sell command.
# This may be useful to prevent players accidentally selling named items.
```